pkg/storkctl/clusterdomainupdate.go

```diff
@@ -0,0 +1,195 @@
+				}
+				clusterDomainUpdate := &storkv1.ClusterDomainUpdate{
+					ObjectMeta: meta.ObjectMeta{
+						Name:      name,
+						Namespace: "kube-system",
+					},
+					Spec: storkv1.ClusterDomainUpdateSpec{
+						ClusterDomain: clusterDomainName,
+						Active:        true,
+					},
+				}
+				_, err := k8s.Instance().CreateClusterDomainUpdate(clusterDomainUpdate)
+				if err != nil {
+					util.CheckErr(fmt.Errorf("Failed to activate cluster domain %v: %v", clusterDomainName, err))
+					return
+				}
+			}
+		},
+	}
+	activateClusterDomainCommand.Flags().BoolVarP(&allClusterDomains, "all", "a", false, "Activate all inactive cluster domains")
+	activateClusterDomainCommand.Flags().StringVar(&nameClusterDomainUpdate, "name", "", "Name for the activate cluster domain action")
+
+	return activateClusterDomainCommand
+}
+
+func newDeactivateClusterDomainCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	var nameClusterDomainUpdate string
+	deactivateClusterDomainCommand := &cobra.Command{
+		Use:     clusterDomainSubcommand,
+		Aliases: clusterDomainAliases,
+		Short:   "Deactivate a cluster domain",
+		Run: func(c *cobra.Command, args []string) {
+			if len(args) == 1 {
+				name := uuid.New()
+				if len(nameClusterDomainUpdate) > 0 {
+					name = nameClusterDomainUpdate
+				}
+				clusterDomainName := args[0]
+				clusterDomainUpdate := &storkv1.ClusterDomainUpdate{
+					ObjectMeta: meta.ObjectMeta{
+						Name:      name,
+						Namespace: "kube-system",
+					},
+					Spec: storkv1.ClusterDomainUpdateSpec{
+						ClusterDomain: clusterDomainName,
+						Active:        false,
+					},
+				}
+				_, err := k8s.Instance().CreateClusterDomainUpdate(clusterDomainUpdate)
+				if err != nil {
+					util.CheckErr(fmt.Errorf("Failed to deactivate cluster domain %v: %v", clusterDomainName, err))
+					return
+				}
+			} else {
+				util.CheckErr(fmt.Errorf("Exactly one cluster domain name needs to be provided to the deactivate command"))
+				return
+			}
+		},
+	}
+	deactivateClusterDomainCommand.Flags().StringVar(&nameClusterDomainUpdate, "name", "", "Name for the deactivate cluster domain action")
+	return deactivateClusterDomainCommand
+}
+
+func newGetClusterDomainUpdateCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	getClusterDomainUpdateCommand := &cobra.Command{
+		Use:     clusterDomainUpdateSubcommand,
+		Aliases: clusterDomainUpdateAliases,
+		Short:   "Get cluster domain updates",
+		Run: func(c *cobra.Command, args []string) {
+			cdStatuses := new(storkv1.ClusterDomainUpdateList)
+			var err error
+			if len(args) > 0 {
+				for _, clusterID := range args {
+					cds, err := k8s.Instance().GetClusterDomainUpdate(clusterID, "kube-system")
+					if err != nil {
+						util.CheckErr(err)
+						return
+					}
+					cdStatuses.Items = append(cdStatuses.Items, *cds)
+				}
+			} else {
+				cdStatuses, err = k8s.Instance().ListClusterDomainUpdates("kube-sytem")
```

Comment:
There is a typo here `kube-sytem`. I'm guessing a unit testing is missing for this code path else it would have failed.